js/Events/evaluationEvents.js

```diff
@@ -0,0 +1,14 @@
+const addEventsEvaluations = function() {
+    const buttonNew = document.getElementById('button2');
+
+    const newButtonEvent = function(event){
```

Comment:
Perhaps the name should be a little bit more specific :)

Reply:
Do you reefer  to  "button2"

Reply:
buttonNew, button2 and newButtonEvent